### PR TITLE
Task: Program For Slice

### PR DESCRIPTION
**Pull request by atharvaskadam**
@aasisodiya Check This One Tried Slice

### CODE REVIEW

File: slice/main.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	fmt.Println("Appended slice", t)
+
+	u := make([]string, len(t))
+	copy(u, t)
+	fmt.Println("cpy", u)
+
+	l := s[2:5]
+	fmt.Println("Sli", l)
+
+	f := s[0:4]
```
Comment (aasisodiya) [inline slice/main.go:48]: What is this coding doing here exactly? can you explain that? if yes - good, else focus on understanding that first 😊

File: slice/main.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+	fmt.Println("len:", len(s))
+
+	fmt.Println("----------------------------")
+	s = append(s, "d")
+	s = append(s, "e", "f")
+	fmt.Println("appended:", s)
+
+	c := make([]string, len(s))
+	copy(c, s)
```
Comment (aasisodiya) [inline slice/main.go:23]: Do you know what `copy()` does in this case? If not study that in depth

File: slice/main.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	fmt.Println("Set", t)
+	fmt.Println("Get", t[3])
+	fmt.Println("len", len(t))
+	t = append(t, "Nissan")
+	t = append(t, "Skoda", "Ferrari")
+	fmt.Println("Appended slice", t)
+
+	u := make([]string, len(t))
+	copy(u, t)
+	fmt.Println("cpy", u)
```
Comment (aasisodiya) [inline slice/main.go:43]: Can you tell me why we use `make()` ?

**Review by aasisodiya — APPROVED** [review]

File: slice/main.go (hunk 1 above)
Comment (atharvaskadam) [reply slice/main.go:48]: Here L Is The Variable Which Is Declared Using Shorthand Declaration Method And s[ ] Is The Slice In Which There Are Elements And s[2:5] Will Give You The Elements At That Location Without The Last Index Position That Is [2:5] Means The Elements  At  Index 2 3 4 We Get.

Comment (atharvaskadam) [reply slice/main.go:48]: Same Is Done For f:=[0:4]

File: slice/main.go (hunk 2 above)
Comment (atharvaskadam) [reply slice/main.go:23]: Here we Have created The empty slice c of the same length as Of s and copied it into c from s

File: slice/main.go (hunk 3 above)
Comment (atharvaskadam) [reply slice/main.go:43]: To create an empty slice we Use Make Which Is Keyword.

**Pull request merged**